src/components/AdminDashboard/adminSection.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const AdminSection = (props) => {
+  const {
+    title,
+    linkTo,
+    createLinkTo,
+    config: {
+      isLoading,
+      hasError,
+      error,
+      resultCount,
+    },
+  } = props;
+
+  const disabledLink = text => <a className="disabled">{text}</a>;
+  const validLink = text => <Link to={linkTo}>{text}</Link>;
+  const sectionLink = () => {
+    if (isLoading) {
+      return disabledLink(`${title} are loading...`);
+    } else {
+      if (hasError) {
+        return disabledLink(error);
+      } else {
+        if (resultCount === 0) {
+          return  disabledLink(`No ${title} created`);
+        } else {
+          return validLink(`View All ${title} (${resultCount})`);
```

Comment:
There's no need for this nesting/the else statements;
```
const sectionLink = () => {
  if (isLoading) return disabledLink(...);
  if (hasError) return disabledLink(...);
  if (resultCount === 0) return disabledLink(...);
  return validLink(...);
}
```
will work fine because of the early returns.

Reply:
Very true. Ive updated this